modules/perc-distribution-tree/src/main/resources/distribution/rxconfig/Installer/install.xml

```diff
@@ -547,7 +553,19 @@
         <exclude name="ObjectStore/**"/>
         <exclude name="jetty/base/logs"/>
         <exclude name="rxconfig/Installer/*.bat"/>
-        <exclude name="ObjectStore/psx_cerff*.xml"/>
+        <exclude name="ObjectStore/psx_cerffAutoIndex.xml"/>
```

Comment:
557-568 We don't want to outright exclude these.  We need to overwrite them if they are present on upgrade, and not install them if they are not present.